package.json

```diff
@@ -11,7 +11,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "body-parser": "^1.18.2",
-    "express": "^4.16.2",
+    "express": "^4.17.3",
```

Comment:
the CI tests are saying that it appears to be an inconsistency between package-lock.json and the package.json